README.md

```diff
@@ -78,10 +84,15 @@ These steps will help you configure the layers correctly:
   * Java:
     * RequestHandler implementation: `com.newrelic.java.HandlerWrapper::handleRequest`
     * RequestStreamHandlerWrapper implementation: `com.newrelic.java.HandlerWrapper::handleStreamsRequest`
+  * .NET: This step is not required.
 4. Add these environment variables to your Lambda console:
   * NEW_RELIC_ACCOUNT_ID: Your New Relic account ID
   * NEW_RELIC_LAMBDA_HANDLER: Path to your initial handler.
   * NEW_RELIC_USE_ESM: For Node.js handlers using ES Modules, set to `true`.
+  * CORECLR_ENABLE_PROFILING (.NET only): 1
+  * CORECLR_PROFILER (.NET only): {36032161-FFC0-4B61-B559-F6C5D41BAE5A}
```

Comment:
At this point we're in a public PR, but is it expected for this value to be exposed?

Reply:
It does look like a key but that's the GUID that identifies our Agent. It's meant to be public. :) [More details](https://docs.newrelic.com/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables/) if you're curious.